.rubocop_todo.yml

```diff
@@ -971,6 +971,7 @@ Style/PerlBackrefs:
 Style/PredicateName:
   Exclude:
     - 'spec/**/*'
+    - 'src/api/app/controllers/webui/kiwi/images_controller.rb'
```

Comment:
For pundit makes sense for sure, in controllers maybe not that much, at least the `?` part. Rubocop doesn't care about Rails, is just a static ruby code analyzer..., so for us is maybe better to skip the rule in controllers or in the place it happens. An action called `outdated?` is a little bit strange but if the method is not an action then maybe is ok.